go/vt/vtgate/planbuilder/insert.go

```diff
@@ -232,6 +232,9 @@ func modifyForAutoinc(ins *sqlparser.Insert, eins *engine.Insert) error {
 func swapBindVariables(rows sqlparser.Values, colNum int, baseName string) (sqltypes.PlanValue, error) {
 	pv := sqltypes.PlanValue{}
 	for rowNum, row := range rows {
+		if _, ok := row[colNum].(*sqlparser.Default); ok {
```

Comment:
One minor nit: It's better to move this code into `modifyForAutoinc` because it's now specific to that context. Otherwise, there's a risk that someone may accidentally reuse this function for other purposes.

Reply:
I agree with this conceptually. I was just trying to not add another loop for the sake of performance. Should I rename this function instead?

Reply:
Rename sounds good.

Reply:
Sorry, I think I misread what you suggested. If I move this code into modifyForAutoinc, I think we'd like this function to be deleted as well. Is that correct?

Reply:
Right. I think this function was previously reused, but it looks like I inlined it for the other use case.